telemetry/record.go

```diff
@@ -35,6 +35,9 @@ var (
 		"V": func() proto.Message {
 			return &protos.Payload{}
 		},
+		"connectivity": func() proto.Message {
```

Comment:
This is should be disabled by default and we can enable it with a feature flag.

Reply:
dispatching connectivity events will be disabled by default. It will only be sent to configured dispatchers if its configured. Example configuration integration test which [enables dispatching connectivity](https://github.com/teslamotors/fleet-telemetry/blob/connectivity_publisher/test/integration/config.json#L49)